app/controllers/concerns/twilio_config.rb

```diff
@@ -80,7 +87,7 @@ def twilio_creds
   end
 
   def user(mobile = nil)
-    @user ||= User.find_by_mobile(mobile || params["From"])
+    @user = User.find_by(mobile: mobile.presence || params["From"])
```

Comment:
what does `find_by(nil)` and `find_by('')` do? Just want to make sure it returns no users!

Reply:
Ideally if there are user records with nil or empty values then it will return those records else it will return no record.